src/hardware/vga_attr.cpp

```diff
@@ -213,93 +241,119 @@ void write_p3c0(io_port_t, io_val_t value, io_width_t)
 				break;
 			case M_VGA:
 			case M_LIN8:
-				vga.config.pel_panning=(val & 0x7)/2;
+				vga.config.pel_panning = (val & 0x7) / 2;
 				break;
 			case M_LIN16:
-			default:
-				vga.config.pel_panning=(val & 0x7);
+			default: vga.config.pel_panning = (val & 0x7);
 			}
-			if (machine==MCH_EGA)
-				// On the EGA panning can be programmed for every scanline:
+			if (machine == MCH_EGA) {
+				// On the EGA panning can be programmed for
+				// every scanline:
 				vga.draw.panning = vga.config.pel_panning;
-			/*
-				0-3	Indicates number of pixels to shift the display left
-					Value  9bit textmode   256color mode   Other modes
-					0          1               0              0
-					1          2              n/a             1
-					2          3               1              2
-					3          4              n/a             3
-					4          5               2              4
-					5          6              n/a             5
-					6          7               3              6
-					7          8              n/a             7
-					8          0              n/a            n/a
-			*/
+			}
+			//      0-3	Indicates number of pixels to shift the
+			// display left Value  9bit textmode   256color mode
+			//  Other modes 0          1               0 0 1 2 n/a 1
+			//              2          3               1 2 3 4 n/a 3
+			//              4          5               2 4 5 6 n/a 5
+			//              6          7               3 6 7 8 n/a 7
+			//              8          0               n/a     n/a
 			break;
-		case 0x14:	/* Color Select Register */
+
```

Comment:
(just a question - is the comment block still OK in this new wrapped-form?)

Reply:
Probably not, but it was screwed up to begin with. To be cleared up using my VGA book during my next refactoring exercise 😄